File: VICSApp/CoreSystem/DHCD/FormUI/Meeting/DelegateUpdate.cs
using System;
using System.Data;
using System.Windows.Forms;
using Microsoft.VisualBasic;
using Microsoft.VisualBasic.CompilerServices;

namespace pmDHCD
{
    public partial class DelegateUpdate
    {
        private int updatedelegatecode;

        private void Button2_Click(object sender, EventArgs e)
        {
            Close();
        }

        private void DelegateUpdate_Load(object sender, EventArgs e)
        {
            MaskedTextBox1.Text = My.MyProject.Forms.Mainform.workingmeeting;
            MaskedTextBox2.Text = updatedelegatecode.ToString();
            var dt = new DataTable();
            try
            {
                dt = My.MyProject.Forms.Mainform.BenlyDal.Delegate_getlist(My.MyProject.Forms.Mainform.workingmeeting, updatedelegatecode, "");
            }
            catch (Exception ex)
            {
                Interaction.MsgBox("Lỗi : " + ex.Message);
                return;
            }

            MaskedTextBox3.Text = Conversions.ToString(dt.Rows[0]["IdentityCard"]);
            MaskedTextBox4.Text = Conversions.ToString(dt.Rows[0]["Delegatename"]);
            MaskedTextBox5.Text = Conversions.ToString(dt.Rows[0]["DelegateAddress"]);
        }

        public DelegateUpdate(int delegatecode)
        {
            updatedelegatecode = delegatecode;
            InitializeComponent();
            _Button1.Name = "Button1";
            _Button2.Name = "Button2";
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            try
            {
                My.MyProject.Forms.Mainform.BenlyDal.Delegate_update(My.MyProject.Forms.Mainform.workingmeeting, updatedelegatecode, MaskedTextBox4.Text, MaskedTextBox3.Text, MaskedTextBox5.Text);
                Close();
            }
            catch (Exception ex)
            {
                Interaction.MsgBox("Lỗi : " + ex.Message);
            }
        }

        private void DelegateUpdate_KeyUp(object sender, KeyEventArgs e)
        {
            if (e.KeyCode == Keys.Escape)
            {
                Close();
            }
        }
    }
}

File: VICSApp/CoreSystem/DHCD/FormUI/Meeting/DelegateUpdate.vb
Public Class DelegateUpdate
    Dim updatedelegatecode As Integer
    Private Sub Button2_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button2.Click
        Me.Close()
    End Sub

    Private Sub DelegateUpdate_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        MaskedTextBox1.Text = Mainform.workingmeeting
        MaskedTextBox2.Text = updatedelegatecode
        Dim dt As New DataTable
        Try
            dt = Mainform.BenlyDal.Delegate_getlist(Mainform.workingmeeting, updatedelegatecode, "")
        Catch ex As Exception
            MsgBox("Lỗi : " + ex.Message)
            Exit Sub
        End Try
        MaskedTextBox3.Text = dt.Rows(0).Item("IdentityCard")
        MaskedTextBox4.Text = dt.Rows(0).Item("Delegatename")
        MaskedTextBox5.Text = dt.Rows(0).Item("DelegateAddress")
    End Sub
    Public Sub New(ByVal delegatecode As Integer)
        Me.updatedelegatecode = delegatecode
        InitializeComponent()
    End Sub

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        Try
            Mainform.BenlyDal.Delegate_update(Mainform.workingmeeting, updatedelegatecode, MaskedTextBox4.Text, MaskedTextBox3.Text, MaskedTextBox5.Text)
            Me.Close()
        Catch ex As Exception
            MsgBox("Lỗi : " + ex.Message)
        End Try
    End Sub

    Private Sub DelegateUpdate_KeyUp(ByVal sender As System.Object, ByVal e As System.Windows.Forms.KeyEventArgs) Handles MyBase.KeyUp
        If e.KeyCode = Keys.Escape Then
            Me.Close()
        End If

    End Sub
End Class